### ISSUE TITLE
RNAfusion: cleanup options

### ISSUE THREAD

**Issue by fevac** [Clinical-Genomics/cg#1959]
## Description

- Remove CLI options that are no longer relevant (including --use-nextflow)

## Suggested solution

*Write here if there is a suggested way to solve the issue*

## This can be closed when

*Describe what needs to be done for this issue to be closed*

## Blocked by

*If there are any blocking issues/prs/things in this or other repos. Please link to them.*

**Comment by fevac** [comment]
- Part one: https://github.com/Clinical-Genomics/cg/pull/2407
- We will keep with-nextflow option for now

**Comment by fevac** [comment]
Done in https://github.com/Clinical-Genomics/cg/pull/2407
